doc/changelog/07-commands-and-options/10734-add-field-no-add-ring.rst

```diff
@@ -0,0 +1,6 @@
+- **Changed:**
+  The :cmd:`Add Field` no longer declares ring structures. This was
```

Comment:
Is there a backwards-compatible suggested fix?  I think it would be nice to describe it here.

Reply:
For now, coq-community/math-classes#81 suggests that preparing a backward-compatible fix is not easy. But a solution should be found so that there is one.

Reply:
Maybe this PR should be implemented using an attribute on `Add Field`?

Reply:
Are unrecognized attributes a no-op?  If not (maybe even if so), we should also get a flag that can be set in the compat file.

Reply:
Unrecognized attributes are a warning which errors by default.

Reply:
> Unrecognized attributes are a warning which errors by default.

Since 8.11.

Reply:
That being said, I generally support attributes and flags going hand in hand.

Reply:
Is it needed in math-classes that the `Add Ring F2` rather than, say, `Add Ring F2_ring`?